dpgen/tools/relabel.py

```diff
@@ -131,6 +147,10 @@ def create_init_tasks(target_folder, param_file, output, fp_json, verbose = True
                     fp_params = fp_jdata['fp_params']
                     user_input = False
                 make_pwscf('.', fp_params, mass_map, fp_pp_files, fp_pp_files, user_input)
+            elif fp_style == "abacus/scf":
+                if "user_fp_params" not in fp_jdata:
+                    raise RuntimeError("'user_fp_params' key and its value are necessary for 'abacus/scf' fp_style.")
+                make_abacus_pw_scf(".", fp_params, mass_map, fp_pp_files, fp_pp_files)
             elif fp_style == 'siesta':
```

Comment:
Did you try `relabel.py` with Abacus?  I didn't see the unit tests for this part.